src/cypress/plugins/index.js

```diff
@@ -0,0 +1,30 @@
+
+/// <reference types="cypress" />
+// ***********************************************************
+// This example plugins/index.js can be used to load plugins
+//
+// You can change the location of this file or turn off loading
+// the plugins file with the 'pluginsFile' configuration option.
+//
+// You can read more here:
+// https://on.cypress.io/plugins-guide
+// ***********************************************************
+
+// This function is called when a project is opened or re-opened (e.g. due to
+// the project's config changing)
+
+/**
+ * @type {Cypress.PluginConfig}
+ */
+ const cucumber = require('cypress-cucumber-preprocessor').default
+ let tradedetails
```

Comment:
This is redundant

Reply:
I removed that line but failed to commit, will do that in next commit